htdocs/product/class/product.class.php

```diff
@@ -1592,6 +1591,9 @@ public function update($id, $user, $notrigger = 0, $action = 'update', $updatety
 			$sql .= ", fk_price_expression = ".($this->fk_price_expression != 0 ? (int) $this->fk_price_expression : 'NULL');
 			$sql .= ", fk_user_modif = ".($user->id > 0 ? $user->id : 'NULL');
 			$sql .= ", mandatory_period = ".($this->mandatory_period);
+			if (getDolGlobalString('PRODUCT_USE_CUSTOMER_PACKAGING') && !empty($this->packaging)) {
+				$sql .= ", packaging = ".($this->packaging);
```

Comment:
I believe that you need to cast ($this->packaging) so that it passes the security tests.